guides/doc-Content_Management_Guide/topics/Importing_Red_Hat_Content.adoc

```diff
@@ -51,7 +51,7 @@ These policies are not available if a {SmartProxy} was installed or updated with
 
 === Changing the Default Download Policy
 
-You can set the default download policy that {Project} applies to repositories that you create in all organizations. {Project} uses separate settings for Red Hat and {customrepos}. Changing the default value does not change existing settings.
+You can set the default download policy that {Project} applies to repositories that you create in all organizations. {Project} uses separate settings for Red Hat and {customrepo}. Changing the default value does not change existing settings.
```

Comment:
In this sentence, the current upstream value of "repository" does not make sense. {customrepo} can be set to "non-Red Hat repository" upstream to avoid this confusion.

Reply:
It is used in many places, so I added ifevals for this context. Everywhere else it works OK. It's only when there is the direct contrast between RH and non-RH.